packages/web-core/src/services/ConnectService.ts

```diff
@@ -516,10 +512,10 @@ export class ConnectService {
         isUserVerifyingPlatformAuthenticatorAvailable: canUsePasskeys,
         isConditionalMediationAvailable: canUseConditionalUI,
         clientEnvHandle: maybeClientHandle ?? undefined,
+        visitorId: currentVisitorId,
```

Comment:
For the backend it' easier to handle if visitorId is part of clientinformation (it is also more similar to clientEnvHandle then)